integration-tests/emulated/src/tests/spiritnet/unpaid_execution.rs

```diff
@@ -80,3 +81,55 @@ fn test_unpaid_execution_to_spiritnet() {
 		assert_eq!(Polkadot::events().len(), 0);
 	});
 }
+
+#[test]
+fn test_unpaid_execution_from_polkadot_to_spiritnet() {
+	MockNetworkPolkadot::reset();
+
+	let sudo_origin = <Polkadot as RelayChain>::RuntimeOrigin::root();
+	let parachain_destination: VersionedMultiLocation = Junctions::X1(Junction::Parachain(spiritnet::PARA_ID)).into();
+
+	let weight_limit = WeightLimit::Unlimited;
+	let check_origin = None;
+
+	let xcm = VersionedXcm::from(Xcm(vec![UnpaidExecution {
```

Comment:
Can we add another instruction after this one, to make sure the operation goes through? I guess a system remark is fine.

Reply:
yeah I could not use system remark or any other extrinsic. We only whitelisted did related extrinsic. So I adjusted the testcase in such a way, that the sovereign account of polkadot wants to create a did: [5699a44](https://github.com/KILTprotocol/kilt-node/pull/603/commits/5699a44b4c96a8b62e63b792a5a76f9cddf33529)